osu.Game/Rulesets/UI/ModSwitchTiny.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Utils;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osu.Game.Rulesets.Mods;
+using osuTK;
+using osuTK.Graphics;
+
+#nullable enable
+
+namespace osu.Game.Rulesets.UI
+{
+    public class ModSwitchTiny : CompositeDrawable
+    {
+        public BindableBool Active { get; } = new BindableBool();
+
+        private readonly IMod mod;
+
+        private readonly Box background;
+        private readonly OsuSpriteText acronymText;
+
+        private Color4 activeForegroundColour;
+        private Color4 inactiveForegroundColour;
+
+        private Color4 activeBackgroundColour;
+        private Color4 inactiveBackgroundColour;
+
+        public ModSwitchTiny(IMod mod)
+        {
+            this.mod = mod;
+            Size = new Vector2(73, 30);
```

Comment:
I think I'd like _something_ to be made into a constant here, and it might be best for this `30` value. That way, if you need a particular size of the icon to fit surrounding elements (e.g. in the results screen panels) you'd do something like `Scale = 16 / ModSwitchTiny.DEFAULT_HEIGHT`?

Reply:
Probably fine to expose when we need it? It's only used in this one place for now, so seems okay by my eyes.

Reply:
To a degree I will actually want to scale those to match neighbouring text, for instance, so I've applied this as proposed (to both sizes) as the way I had it on my working branch was a bit hacky anyhow.